### DISCUSSION TITLE
Is midi.js dead? Are there alternatives?

### DISCUSSION THREAD

**Discussion by repenning** [djipco/webmidi#167] [General]
It appears that MIDI.js is not a very active repository. Last substantial updates are several years old. We are using MIDI.js as part of AgentCubes (a programming environment for kids). MIDI.js works ok in some browsers, e.g., Chrome, but is severely broken in others, e.g., Safari. Is MIDI.js essentially dead and if so, are there recommended alternatives?

**Comment by djipco** [comment]
I am transferring this issue to the "Discussions" section.

**Comment by djipco** [comment]
I have never used MIDI.js but it does look like abandonware (no updates in 7 years). Perhaps somebody here knows about the status of this project?
